src/test/fuzz/rbf.cpp

```diff
@@ -113,15 +115,13 @@ FUZZ_TARGET(package_rbf, .init = initialize_package_rbf)
     LIMITED_WHILE(fuzzed_data_provider.ConsumeBool(), NUM_ITERS)
     {
         // Make sure txns only have one input, and that a unique input is given to avoid circular references
```

Comment:
I understand this comment as explaining why a random number of `vin`s of `parent` generated from fuzz data are overwritten with one input. Time to change/remove?